Testing/Functional/Products/TestPlans/teams.testplan.yaml

```diff
@@ -62,10 +50,10 @@ TestPlan:
           - Command: Set-CsTeamsMeetingPolicy
             Splat:
               AutoAdmittedUsers: EveryoneInCompany
-            Identity: Global
+              Identity: Global
         Postconditions: []
         ExpectedResult: false
-      - TestDescription: Dialin not bypass lobbyEveryone not auto admitted user
+      - TestDescription: MS.TEAMS.1.3v1 Compliant - Dialin not bypass lobbyEveryone not auto admitted user
```

Comment:
Misspell in the TestDescription (add ; after lobby)

Reply:
Corrected